central/postgres/pruning.go

```diff
@@ -35,3 +40,27 @@ func PruneClusterHealthStatuses(ctx context.Context, pool *postgres.DB) {
 		log.Errorf("failed to prune cluster health statuses: %v", err)
 	}
 }
+
+func getOrphanedAlertIDs(ctx context.Context, pool *postgres.DB) ([]string, error) {
+	var ids []string
+	rows, err := pool.Query(ctx, getAllOrphanedAlerts)
```

Comment:
Do we want to give this a little more time for now?

Reply:
good point, will make it 5 minutes? This one seems important to run potentially